src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import db.Storage;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.DataConverter;
+import service.FileReader;
+import service.ReportGenerator;
+import service.ShopService;
+import service.impl.DataConverterImpl;
+import service.impl.FileReaderImpl;
+import service.impl.FileWriterImpl;
+import service.impl.OperationStrategyImpl;
+import service.impl.ReportGeneratorImpl;
+import service.impl.ShopServiceImpl;
+import service.operation.BalanceOperation;
+import service.operation.OperationHandler;
+import service.operation.PurchaseOperation;
+import service.operation.ReturnOperation;
+import service.operation.SupplyOperation;
+import strategy.OperationStrategy;
+
+public class Main {
+    private static final String REPORT_TO_READ_FILE = "src/main/java/resources/reportToRead.csv";
```

Comment:
Create a system resources directory. It has an incorrect level 
```suggestion
    private static final String REPORT_TO_READ_FILE = "src/main/resources/reportToRead.csv";
```